oauth2-server/pom.xml

```diff
@@ -80,4 +87,33 @@
 
   </dependencies>
 
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.sonatype.plugins</groupId>
+        <artifactId>jarjar-maven-plugin</artifactId>
+        <executions>
+          <execution>
+            <phase>package</phase>
+            <goals>
+              <goal>jarjar</goal>
+            </goals>
+            <configuration>
+              <includes>
+                <include>jjwt</include>
+                <include>io.jsonwebtoken:jjwt</include>
+              </includes>
+              <rules>
+                <rule>
+                  <pattern>io.jsonwebtoken.**</pattern>
+                  <result> com.clouway.security.internal.jwt.@1</result>
```

Comment:
there is a space in the tag and I'm not sure whether it's a problem for the parser or not.